packages/core/src/experimental/index.ts

```diff
@@ -0,0 +1,60 @@
+export {
+  useSession as useSession_unstable,
```

Comment:
Why not use the `_experimental` suffix instead of `_unstable`?

Reply:
I did it because it's how React usually name their prefix their experimental functions